### ISSUE TITLE
International Website 

### ISSUE THREAD

**Issue by marcoherbst** [evercam/evercam.io#13]

**Comment by marcoherbst** [comment]
evercam.io/uk
evercam.co.nz
evercam.sg
evercam.ie

**Comment by asimism** [comment]
I don't think we can use wordpress multisite feature to make this happen.  There is a lot of complexity in terms of Zoho CRM forms and how we capture the leads and other Google tracking and SEO plugins in place. Multisite shares the plugins across different sites and most plugins are not enabled for multisite functionality therefore it would be almost impossible to manage the plugins functionality across different regional sites. I think we will be better of having separate WordPress installations for every region.

**Comment by asimism** [comment]
Another thing, in multisite setup any content created like a page or post will only be created in a specific site not across all sites in the network.
